generator/resources/witx_module.mustache

```diff
@@ -0,0 +1,29 @@
+package swam
+package wasi
+
+import Types._
+import Header._
+import cats.Applicative
+import cats.effect._
+import swam.runtime.formats._
+import swam.runtime.formats.DefaultFormatters._
+import cats.effect.IO
+import swam.runtime.Memory
+import swam.runtime.imports.annotations.{effect, effectful, module, pure}
+
+
+
+@module
+abstract class Module[@effect F[_]](val mem: Memory[IO])(implicit F: Applicative[F])
+{
+    val name = "{{{moduleName}}}"
+
+    def tryToExecute(a: => errnoEnum.Value) = {
```

Comment:
This method looks like it's WASI specific (errno and friends), and won't make sense outside of this specific use case. I am not sure it is the appropriate place to have this.